components/datatypes/productlistitem.schema.json

```diff
@@ -56,6 +56,77 @@
           "title": "Price total",
           "type": "number",
           "description": "The total price for the product line item."
+        },
+        "xdm:itemsInProductList": {
+          "title": "Items in Product List",
+          "type": "array",
+          "items": { "type": "string" },
+          "description": "An array of all product IDs/SKUs in the list of products. Used outside of a shopping cart."
+        },
+        "xdm:productImpression": {
+          "title": "Product Impression",
+          "$ref": "https://ns.adobe.com/xdm/data/measure",
+          "description": "Counted any time a product thumbnail or summary view has been displayed, aside from the product detail page itself or as an item in a cart."
+        },
+        "xdm:skuVariant": {
+          "title": "Product SKU Variant",
+          "type": "string",
+          "description": "Tracks if a product has alternate SKUs or IDs, such as different SKUs for color variants."
+        },
+        "xdm:brand": {
+          "title": "Product Brand",
+          "type": "string",
+          "description": "Tracks the brand of the individual product."
+        },
+        "xdm:category": {
```

Comment:
Can you please get in touch with Rachel/Prabhakar to see if you can use the standard product classes/mixins to populate category and sub-category. They are coming up with a bunch of Product mixins.